app/angular/src/server/angular-cli_config.ts

```diff
@@ -90,6 +108,9 @@ export function getAngularCliWebpackConfigOptions(dirToSearch: Path) {
   const tsConfigPath = path.resolve(dirToSearch, projectOptions.tsConfig) as Path;
   const tsConfig = getTsConfigOptions(tsConfigPath);
   const budgets = projectOptions.budgets || [];
+  const scripts = projectOptions.scripts || [];
+  const outputPath = projectOptions.outputPath || 'dist/storybook-angular';
```

Comment:
what's the output path?

Reply:
`outputPath` is needed by `angular.json` - more specifically it is needed by the function we call in `@angular-devkit` for receiving the default angular webpack config.

Storybook does not need it and usually it is already set in `angular.json`. If it is not set then storybook throws an error that it can't load the angular cli configuration